docker-plugin/src/main/java/io/micronaut/gradle/docker/MicronautDockerPlugin.java

```diff
@@ -305,7 +306,9 @@ private TaskProvider<NativeImageDockerfile> configureNativeDockerBuild(Project p
         TaskProvider<DockerBuildImage> dockerBuildTask = tasks.register(adaptTaskName("dockerBuildNative", imageName), DockerBuildImage.class, task -> {
             task.setGroup(BasePlugin.BUILD_GROUP);
             task.setDescription("Builds a Native Docker Image using GraalVM (image " + imageName + ")");
-            task.getInputs().files(prepareContext);
+            task.getInputs().files(prepareContext)
+                    .withPropertyName("PrepareDockerContext.outputs")
```

Comment:
since this is an input, i would recommend naming it something like `preparedDockerContext`.